lib/kino_db/connection_cell.ex

```diff
@@ -159,6 +162,9 @@ defmodule KinoDB.ConnectionCell do
         "sqlite" ->
           ~w|database_path|
 
+        "duckdb" ->
+          []
```

Comment:
Don't we always need the path?

```suggestion
          ~w|database_path|
```

Reply:
It is optional, if unset it runs on memory.

Reply:
I see, in that case I think we should generate the code without `path` if it is empty, wdyt?

Reply:
sure, making it more explicit. No problem!